migrations/20221012185940_add_repo_sync_type_groups.up.sql

```diff
@@ -0,0 +1,95 @@
+BEGIN;
```

Comment:
My comments PR is going to merge first so IMO we should rename this migration so it runs after that one. Agree @patrickdevivo ?